src/components/dashboard/sidebar-item.tsx

```diff
@@ -11,31 +12,36 @@ export const SidebarItem = (props: SidebarItemProps) => {
 
   const { pathname } = useRouter()
   return (
-    <Flex
-      h="46px"
-      align="center"
-      pl="15"
-      gap="18px"
-      cursor="default"
-      data-active={dataAttr(pathname === item.route)}
-      _hover={{
-        bg: 'whiteAlpha.200',
-      }}
-      _active={{
-        bg: 'whiteAlpha.200',
-      }}
-      transition="background .2s ease"
-      role="group"
-      {...rest}
-    >
-      <Icon as={item.icon} boxSize="6" /> <span>{item.label}</span>
-      <NavIndicator
-        display="none"
-        _groupActive={{
-          display: 'inherit',
+    <NextLink href={item.route} passHref>
```

Comment:
We should use a LinkOverlay here. It hurts accessibility to have an anchor tag over divs